### PR TITLE
Update documentation dropdown

### PR DESCRIPTION
**Pull request by adrn**
This updates the documentation dropdown menu for the new release.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,9 +49,9 @@
 						<a href="http://docs.astropy.org">Documentation</a>
 						<div class="dropdown-content">
 							<ul>
-								<li><a href="http://docs.astropy.org" target="_blank">Current Release</a></li>
+								<li><a href="http://docs.astropy.org" target="_blank">Current Release (3.1)</a></li>
 								<li><a href="http://devdocs.astropy.org/" target="_blank">In Development</a></li>
-								<li><a href="https://astropy.readthedocs.io/en/v2.0.9/index.html" target="_blank">v2.0.9</a></li>
+								<li><a href="https://astropy.readthedocs.io/en/v3.0.5/index.html" target="_blank">v3.0.5</a></li>
```
Comment (bsipocz) [inline index.html:54]: Don't update this version to non LTS, but rather 2.0.10

File: index.html [hunk 2]
```diff
@@ -49,9 +49,9 @@
 						<a href="http://docs.astropy.org">Documentation</a>
 						<div class="dropdown-content">
 							<ul>
-								<li><a href="http://docs.astropy.org" target="_blank">Current Release</a></li>
+								<li><a href="http://docs.astropy.org" target="_blank">Current Release (3.1)</a></li>
```
Comment (bsipocz) [inline index.html:52]: I'm -1 on this

**Comment by adrn** [comment]
IMO, we should either go all-in on putting version numbers in the dropdown, or none. I prefer having the version numbers there as a quick way for users to see what is new / old. But I know @bsipocz is going to push back and say that creates the possibility of the numbers being out of sync. If we take that philosophy, then I suggest instead removing v2.0.9 and instead having "current release", "in development", "long-term support" (and link the last one to http://docs.astropy.org/en/lts/).

File: index.html (hunk 1 above)
Comment (adrn) [reply index.html:54]: See comment below. If this is really supposed to be LTS, it should link to http://docs.astropy.org/en/lts/

File: index.html (hunk 2 above)
Comment (adrn) [reply index.html:52]: I think I know why, but can you explain?

**Comment by bsipocz** [comment]
I'm all on board with removing 2.0.9 and listing LTS instead, that actually would make things super consistent as those three are the ones we support.

We've removed listing more versions just a couple of weeks ago as part of their removal from RTD in order to fix the google search results.

**Comment by eteq** [comment]
I like the "words only" solution. Will implement that in astropy/astropy.github.com#286 (and close this once it's in)

**Pull request closed without merging**